### ISSUE TITLE
Lazy tab load on startup

### ISSUE THREAD

**Issue by chaoscreater** [Alex313031/Thorium-Win#171]
Hi there,

In Vivaldi, which is another browser based on Chromium, it's able to do tab lazy load on startup. If you have 10 tabs, it will only load the active tab, instead of loading all 10 tabs. In Thorium, the startup is always slow for me because it has to load a bunch of tabs. Is there a flag or commandline to disable this behaviour? Or could this possibly be implemented as a feature?

**Comment by trlkly** [comment]
If you haven't resolved this already, you may want to look for a Chrome Extension called Sloth.

**Comment by chaoscreater** [comment]
> If you haven't resolved this already, you may want to look for a Chrome Extension called Sloth.

Thanks, I just tried it and it seems to work fine. 

It's an improvement to the issue I have, but it's not really addressing the root issue. When I startup Thorium, it lags for a good 10 seconds or so before I can use it. During this time, it's loading background tabs (not sure the exact count, maybe like 8~10 of them) and I have a total of 400 tabs. Sloth is able to suspend the loaded background tabs, but it doesn't "prevent" them from ever loading in the first place. The root issue is that Thorium loads the background tabs on startup and this shouldn't happen in the first place.

**Comment by Kol9yN** [comment]
Agreed with author, Thorium should have it too as FireFox and WaterFox.
